jest/jestSetup.js

```diff
@@ -51,6 +51,9 @@ jest.mock('react-native', () => {
       resourceURL:
         'file:///private/var/containers/Bundle/Application/4DCD4D2B-F745-4C70-AD74-8E5F690CF593/ZulipMobile.app/',
     };
+    ReactNative.NativeModules.ZLPConstants = {
+      areNotificationsAuthorized: () => Promise.resolve(true),
+    };
   } else if (ReactNative.Platform.OS === 'android') {
```

Comment:
Huh, interesting that this is only needed on iOS (that is, when Jest is pretending we're on iOS), and not Android. Do you have an understanding of why?

Reply:
Oh, I don't think this is actually needed; I just thought it might be convenient for future tests one might write. I forgot to add a corresponding mock for Android, and in the iOS mock I used the wrong name (`ZLPConstants` for `ZLPNotifications`). I think I'll back this out, then if we need it we can always add it.